requirements.txt

```diff
@@ -0,0 +1,2 @@
+protobuf
```

Comment:
Please remove this file as it is not relevant to this PR. Send another for this.